units/stats_task_end.go

```diff
@@ -84,19 +87,7 @@ func (j *collectTaskEndDataJob) Run(ctx context.Context) {
 		}
 	}
 	if j.task == nil {
-		j.AddError(errors.Errorf("Could not find task '%s'", j.TaskID))
-		return
-	}
-
-	if j.host == nil {
-		j.host, err = host.FindOneId(j.HostID)
-		j.AddError(err)
-		if err != nil {
-			return
-		}
-	}
-	if j.host == nil {
-		j.AddError(errors.Errorf("Could not find host '%s'", j.HostID))
+		j.AddError(errors.Errorf("finding task '%s'", j.TaskID))
```

Comment:
The `finding task` convention only applies to `errors.Wrap`/`errors.Wrapf` because they only wrap an existing error with more messages. For base errors (i.e. new errors created by `errors.New`/`errors.Errorf`), they should be written like an error message, like `errors.Errorf("task '%s' not found", j.TaskID)`